pyproject.toml

```diff
@@ -6,8 +6,3 @@ requires = [
   "wheel",
 ]
 build-backend = "setuptools.build_meta"
-
-# ATTENTION: the following section must be kept last in
-# `pyproject.toml` because Travis CI appends one line in
```

Comment:
This mustn't be removed as it's mandatory for the current setuptools-scm setup. The comment will be changed to GHA.